.github/workflows/unit_test.yml

```diff
@@ -57,6 +57,16 @@ jobs:
     - name: Run 
       run: make test-verbose
 
+  gofmt_test:
```

Comment:
`needs: gofmt_test`
is worth to consider that by line above, makes lint check before unit test.

Reply:
Why can't we run the tests in parallel?

These tests are independent and making them serial will slow down the process...